src/Pages/login/Login.tsx

```
@@ -27,27 +27,62 @@ const Login: React.FC<loginUserProps> = ({ userLoggedIn }) => {
         }
     }
 
+function sonarIssuesDemo(password: string, userEmail: string) {
+   
+    const hardcodedPassword = "P@ssw0rd123";
+   
+    let unusedVar = 42;
+  
+    const code = "console.log('Eval is dangerous!')";
+    eval(code);
+
+
+    console.log("User email:", userEmail, "Password:", password);
+
+
+    try {
+        throw new Error("Test error");
+    } catch (e) {
+
```

Throw error instead of swallowing